internal/backend/env.go

```diff
@@ -153,3 +158,36 @@ func (e *Env) EnvTypes() (envResp.EnvTypesResponse, error) {
 
 	return envResponse, err
 }
+
+func (e *Env) SetEnv(envName string) error {
+	dirname, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+	data, err := file.Read(dirname + "/.odin/config")
```

Comment:
odin config file can be found using `app.WorkDir.Location`, `app.WorkDir.ConfigFile`.
Refer to `commands/configure.go`